dataset.ttl

```diff
@@ -535,10 +575,11 @@ data:GalileeBasin a geof:Basin ,
         ischart:Permian,
         ischart:Pennsylvanian,
         ischart:UpperMississippian ;
-        #skos:scopeNote "That the Galilee Basin section in the Koburra Trough extends down into the Upper Mississippian is uncertain."@en ;#
+        # How do we add a note "That the Galilee Basin section in the Koburra Trough extends down into the Upper Mississippian is uncertain."@en ;#

```

Comment:
Hi John, Can you please edit line 578 of your new file to remove the text "#How do we add a note" and replace it with "#skos:scopeNote"  Thanks.

Reply:
@johnmckellar - see https://www.w3.org/TR/swbp-skos-core-guide/#secdocumentation

Reply:
Thanks Kate. I have modified the dataset for all comments and will upload it again.